dns/dns.go

```diff
@@ -17,6 +17,7 @@ type ResponseResult struct {
 	Response []ResponseEntry `json:"response"`
 	Err      string          `json:"error"`
 	Rcode    int             `json:"rcode"`
+	RawMsg   interface{}     `json:"rawmsg"`
```

Comment:
Does this need to be an interface{} - Presumably we know the type here from the library we are importing.